draft-wang-ppm-differential-privacy.md

```diff
@@ -396,8 +407,10 @@ p(S, D1, r) <= exp(EPSILON) * p(S, D2, r) + DELTA
 ~~~
 
 Compared to pure-DP, approximate-DP loses an additive factor of `DELTA` in the
-bound. This is to account for certain DP mechanisms that have some desirable
-properties, but are not pure-DP. See {{mechanisms}} for details.
+bound. This parameter `DELTA` is typically taken to be polynomially small in
```

Comment:
How does one formalize "polynomially small"? Is there a security parameter in some DP definitions?

Is there a different way to word this?

Reply:
I clarified it a bit:

> `DELTA` can intuitively be understood as the probability that a piece of information is leaked, e.g., a Client measurement is leaked, so `DELTA` is typically taken to be polynomially small in the batch size or smaller, i.e., some value much smaller than `1 / batch_size`.